src/pages/Inventory/AddItem.js

```diff
@@ -1,7 +1,119 @@
-import React from "react";
+import React, { Fragment, useState } from "react";
+import withWrapper from "common/withWrapper";
+import Navbar from "common/Navbar";
+import { useHistory } from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+import { useNotification } from "notification";
 
 const AddItem = () => {
-    return <div>I am add itme page</div>;
+    const [data, setData] = useState({
+        name: "",
+        price: "",
+        description: "",
+        date: new Date(),
+        id: Math.floor(Math.random() * 101 + 1),
+    });
+    const history = useHistory();
+    const { triggerNotification } = useNotification();
+
+    const AddItem = (e) => {
```

Comment:
function name should be camelCase and also use `useCallback`

Reply:
Fixed this one